CryptoTechReminderSystem.Main/.env-default

```diff
@@ -5,5 +5,6 @@ HARVEST_USER_AGENT=
 SLACK_CHANNEL_ID=
 HARVEST_BILLABLE_ROLES=
 SLACK_REMINDER_MESSAGE=
+SLACK_PM_REMINDER_MESSAGE=
```

Comment:
The reminder message to be sent to Project Managers of projects with open time entries.